packages/contracts/test/test-utils/wrapper/hardhat.ts

```diff
@@ -0,0 +1,100 @@
+import {DeployOptions, NetworkDeployment} from '.';
+import {BigNumberish, Contract, providers} from 'ethers';
+import {utils} from 'ethers';
+import hre from 'hardhat';
+
+export class HardhatClass implements NetworkDeployment {
```

Comment:
Why are we reimplementing all these functions?
Just to have a custom zksync counterpart?